python_modules/dagster/dagster/core/log.py

```diff
@@ -132,3 +133,9 @@ def critical(self, msg, **kwargs):
 
         See debug()'''
         return self._log('critical', msg, kwargs)
+
+    def __getattr__(self, name):
+        def handler(msg, **kwargs):
+            return self._log(name.lower(), msg, kwargs)
```

Comment:
This seems problematic, especially when combined with above issue. what if they are using a library that doesn't user pythonic naming conventions?

Reply:
we could omit the call to `.lower()` here -- but all that `._log` does is dispatch to the underlying loggers